branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -41,6 +41,12 @@ public boolean hasEntry(String branchName) {
     return getEntryByName(branchName) != null;
   }
 
+  public boolean isEntryDuplicated(String branchName) {
+    val numberOfEntriesForBranchName = rootEntries.flatMap(BranchLayout::collectEntriesRecursively)
```

Comment:
While you're here... I'd extract `rootEntries.flatMap(BranchLayout::collectEntriesRecursively)` to a val (potentially `Lazy` from Vavr?)

Reply:
sure, will do before merge